test/commands/index.test.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+'use strict';
```

Comment:
General note for unit tests: On a clean checkout (after `npm i`) for this branch, tests fail on Windows, for various reasons:
- Some are access permissions (Windows is more restrictive sometimes) - the fs error code to handle is `EPERM`;
- Clean ups (removing directories that are not empty is not authorized - the error code here to handle is `ENOTEMPTY`);
- Differences in how stdout is represented (that's for some tests checking what test-worker should display).